### ISSUE TITLE
Backport REPL patch to 0.11?

### ISSUE THREAD

**Issue by Alaneor** [balderdashy/sails#3390]
There has been a REPL fix applied in afce88c29360f88c634294dd794064f293b38399 which, in addition to other issues, fixes a weird problem where using `instanceof` does not work for built-in objects.

For example, let's assume the following is a sails service, *mysvc*:

```js
exports.test = function test (input) {
  if (! (input instanceof Object)) {
    throw new Error('Not an object')
  }
}
```

If we call this function from `sails console`, the error will be thrown:

```js
// sails console
sails.services.mysvc.test({})
// Error: Not an object
```

I do not think this is expected behaviour. It would be great to have this commit released in 0.11.4 or similar patch-level release. Thanks!

**Comment by sailsbot** [comment]
Thanks for posting, @Alaneor.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
